code/controllers/configuration/sections/mc_configuration.dm

```diff
@@ -0,0 +1,23 @@
+/// Config holder for all things regarding the MC
+/datum/configuration_section/mc_configuration
+	/// Server ticklag
+	var/ticklag = 0.5
+	/// Tick limit % during world Init
+	var/world_init_tick_limit = TICK_LIMIT_MC_INIT_DEFAULT
+	/// Base MC tick rate
+	var/mc_base_tickrate = 1
+	/// Highpop MC tickrate
+	var/mc_highpop_tickrate = 1.1
+	/// MC Highpop enable threshold
+	var/mc_highpop_enable_threshold = 65
+	/// MC Highpop disable threshold
+	var/mc_highpop_disable_threshold = 60
```

Comment:
Could these maybe just be `base_tickrate`, `highpop_tickrate`, etc without the `mc_` at the start?
They'll only be accessible through `var/datum/configuration_section/mc_configuration/mc` anyway so having two `mc`'s in  `GLOB.configuration.mc.mc_base_tickrate` seems a bit redundant.